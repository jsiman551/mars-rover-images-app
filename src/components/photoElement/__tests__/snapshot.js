import { render } from '@testing-library/react';
import PhotoElement from '../index';

it('renders pagination buttons unchanged', () => {
  const { container } = render(<PhotoElement />);
  expect(container).toMatchSnapshot();
});
