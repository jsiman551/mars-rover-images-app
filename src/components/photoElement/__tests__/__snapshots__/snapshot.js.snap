// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders pagination buttons unchanged 1`] = `
<div>
  <div
    class="css-1wheore"
    data-testid="photo-element-container"
  >
    <div
      class="chakra-card css-2dlz7"
    >
      <div
        class="chakra-card__body css-0"
      >
        <img
          class="chakra-image css-0"
          src=""
        />
        <div
          class="chakra-stack css-29dqc6"
        >
          <h2
            class="chakra-heading css-0"
          >
            Photo Description
          </h2>
          <ul
            class="css-tu0njr"
            role="list"
          >
            <li
              class="css-0"
            >
              full test name (test name)
            </li>
            <li
              class="css-0"
            >
              Earth Date: 2020-02-02
            </li>
            <li
              class="css-0"
            >
              Sol Date: 1000
            </li>
          </ul>
          <p
            class="chakra-text css-1ppa63l"
          >
            Id: 500
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
